include/z64player.h

```diff
@@ -762,7 +762,7 @@ typedef struct WeaponInfo {
 #define PLAYER_STATE2_23 (1 << 23)
 #define PLAYER_STATE2_24 (1 << 24)
 #define PLAYER_STATE2_25 (1 << 25)
-#define PLAYER_STATE2_26 (1 << 26)
+#define PLAYER_STATE2_REFLECTION (1 << 26) // Handles Dark Link's reflection
```

Comment:
`PLAYER_STATE2_FLOOR_REFLECTION_ENABLED // Draws a copy of the entire skeleton, rotated into the floor to give the appearance of a floor reflection`

This is not Dark Link's reflection necessarily, its Link's. And you could easily use this for another reflection purpose in a modding scenario, Dark Link doesnt really need to be mentioned.